MainTest.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.Test;
+
+import static org.junit.Assert.*; 
+
+public class MainTest {
+
+@Test
+public void testNothing() {} 
+
+@Test  
+public void testSomeStuff() {
+assertEquals(5,3+2);
```

Comment:
Sie können (und sollten) als ersten Parameter eine aussagekräftige Message angeben, diese ausgegeben, wenn das asssert fehlschlägt, z.B. `assertEquals( "Fehler in Addition", 5, 3+2 );`